query/src/main/scala/filodb/query/ResultTypes.scala

```diff
@@ -49,7 +51,9 @@ object QueryResultType extends Enum[QueryResultType] {
 
 final case class QueryResult(id: String,
                              resultSchema: ResultSchema,
-                             result: Seq[RangeVector]) extends QueryResponse {
+                             result: Seq[RangeVector],
+                             mayBePartial: Boolean = false,
```

Comment:
`isPartial`

Reply:
It is intentionally called `mayBePartial` and not `isPartial`. We can never be sure because the missed results could be empty.

Reply:
> We can never be sure because the missed results could be empty.

That means `mayBePartial` is always true with what you say? In this case if we are specifically targeting not answering queries or returning partial results when shards are not active, should the flag be specific about that instead of `mayBePartial` ?

Reply:
Ok. As discussed, it may be partial if the timeseries partitions required for the query is already loaded.